lang/ui.es-es.json

```diff
@@ -391,7 +391,7 @@
     "description": "Language option text"
   },
   "less-action": {
-    "defaultMessage": "menos",
```

Comment:
Unexpected translation loss here for less-action, more-action, terms-of-use.

Languages to review: es-es, ko, zh-*

Some other languages look like the string was retranslated though not proposing reviewing those.

Reply:
These were all the master strings without a translation where a duplicate had a translation that we'd previously restored in Crowdin after unexpected structural changes to documents earlier in the year. 

I think this loss is expected in the circumstances and hopefully we've taken steps to ensure it won't recur. I've copied the translations from the duplicates to fix.

I'll do another sync to update this PR.

Reply:
Done, looks OK now. There's some changes in the same strings in the other languages where the translator filled in the missing translations—they seem good.